canned/new_apps.sh

```diff
@@ -49,7 +49,7 @@ cat > $APP_FILE << EOF
 		"i": "$CELLID",
 		"name": "User facing cell Name",
 		"queries": [{
-			"query": "select mean(\"used_percent\") from disk",
+			"query": "SELECT mean(\"used_percent\") FROM disk",
```

Comment:
Could you add `FROM \":db:\".\":rp:\".\"` for this query as well?

Reply:
@ischolten Done; missed because the original PR didn't include changes to the script.